alpha/lib/model/DeliveryProfileHttp.php

```diff
@@ -28,6 +37,32 @@ public function buildServeFlavors()
 		
 		return $flavors;
 	}
+
+	/**
+	 * returns whether the delivery profile supports the passed deliveryAttributes in this case seekFrom
+	 * @param DeliveryProfileDynamicAttributes $deliveryAttributes
+	 * @return int
+	 */
+	public function supportsDeliveryDynamicAttributes(DeliveryProfileDynamicAttributes $deliveryAttributes) {
+		$result = parent::supportsDeliveryDynamicAttributes($deliveryAttributes);
+
+		if ($result == self::DYNAMIC_ATTRIBUTES_NO_SUPPORT)
+			return $result;
+
+		if($this->getMaxSize())
+		{
+			foreach($deliveryAttributes->getFlavorAssets() as $flavorAsset)
+			{
+				$flavorSize = $flavorAsset->getSize() * 1024 * 1024; // size in MB
```

Comment:
Flavor asset getSize is in kbytes, so just multiple in 1024 will give the value in MB